test/clients/firewall_auto_test.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require_relative "../../src/clients/firewall_auto.rb"
```

Comment:
@jreidinger we will apply the nasty workaround for SP1, as we can change it for SP2.